src/app/components/hris/employees/employee-profile/accordions/accordion-profile/accordion-profile-address-details/accordion-profile-address-details.component.ts

```diff
@@ -157,7 +157,7 @@ export class AccordionProfileAddressDetailsComponent {
   }
 
   loadPhysicalAddress() {
-    this.locationApiService.getCountries().subscribe({
+  this.locationApiService.countries$?.subscribe({
```

Comment:
Refactor: this entire function can be reverted because in "getCountries" the observable is stored in a private variable. If it's null, a network request is made; otherwise, the cached observable is returned